### ISSUE TITLE
Possible enhancements to the Toggle List feature

### ISSUE THREAD

**Issue by dgbeecher** [glutanimate/cloze-overlapper#3]
I really appreciate that you included the feature to toggle ordered and unordered lists. I think your add-on  does a better job with lists than the "Power Format Pack" add-on, which I've consequentally deleted. However, there are a few ways I think the list feature could be improved:

1. Currently every line of text becomes a list item when a Toggle List hotkey is pressed. But sometimes I only want parts of the text to be formatted as a list. Would it be possible to introduce the following behavior when a Toggle List hotkey is pressed: if no text is selected, toggle list formatting for all lines; but if only some text is selected, then toggle list formatting for only those lines.

    (Because this suggestion requires you to no longer assume that the user wants every line to be a list item as well as a cloze deletion, it might make sense to de-link toggling list formatting and generating overlapping clozes.)

2. I'd love to be able to use the list feature of this add-on even for other note types. It seems this might be possible if you made toggling list formatting and generating overlapping clozes independent commands. That way, the "Reminder: can only generate overlapping clozes ..." warning would only appear when attempting to generate overlapping cloze. Meanwhile, the user could toggle list formatting freely when working with other note types.

Thanks as always for your great work!

**Comment by glutanimate** [comment]
Thanks again for your thoughtful suggestions! I'm glad you like the list toggles. I can see your point, but I think it would be best if the add-on stayed close to its original purpose. These hotkeys were meant more as a convenience feature and not as general purpose formatting tools.

The Power Format Pack might have a few too many features if you're mainly looking for a way to toggle lists, but there are other add-ons like [Power Create lists ordered unordered and intended](https://ankiweb.net/shared/info/1524262201) that use the same exact method as Cloze Overlapper. You should definitely give this one a try. I think it will fit your use case very well.

**Comment by dgbeecher** [comment]
That makes total sense. Thanks for considering it, and for the good advice!

(Also, I realized that the reason I prefered your implementation of list formatting to the other add-ons wasn't actually because of the add-on code itself, but rather because of the card styling: specifically the  "display: inline-block" and "text-align: left" parts. So I might as well use the the other add-ons for list formatting.)
